src/Pact/Types/Hash.hs

```diff
@@ -39,7 +39,9 @@ hash = Hash . ByteArray.convert . Crypto.hashWith Crypto.Blake2b_512
 import Crypto.Hash.Blake2Native
 
 hash :: ByteString -> Hash
-hash bs = let (Right h) = blake2b hashLengthAsBS mempty bs in Hash h
+hash bs = case blake2b hashLengthAsBS mempty bs of
```

Comment:
Just to clarify...I checked with @vaibhavsagar and this use of blake2b is only used for GHCJS builds (see above #if #else).  And it is calling our native haskell blake functions, not the `blake2` library.